units/local-fs.target

```diff
@@ -12,6 +12,7 @@ Description=Local File Systems
 Documentation=man:systemd.special(7)
 DefaultDependencies=no
 Conflicts=shutdown.target
+Wants=local-fs-pre.target
```

Comment:
hmm?

Reply:
note that local-fs-pre.target is documented to be a "passive" unit, i.e. is not pulled in unless something actually wants to be ordered before it.